api/requirements.txt

```diff
@@ -79,7 +79,7 @@ SQLAlchemy==1.3.*
 # does not work with tzlocal 3+.
 transitions
 tzlocal<3.0.0
-weasyprint
+weasyprint==54.0
```

Comment:
Ça mériterait un commentaire pour éviter qu'un futur nettoyage des dépendances ne retire ~ce pin~ cet épinglage. Suggestion : 

    # Pin weasyprint to keep new versions from changing how our PDF files look.